.github/workflows/deploy-to-development.yml

```diff
@@ -96,13 +90,15 @@ jobs:
       - name: Run the scripts
         run: .github/bin/buildmodel
 
-      - name: Merge PR
-        if: contains(github.event.pull_request.labels.*.name, 'automated-build-model')
-        run: .github/bin/mergepr
-
   validate-model:
     needs: build-model
     runs-on: ubuntu-latest
+
+    env:
```

Comment:
So maybe to avoid leaking secrets into steps we don't want we can just put these env vars against the step that needs them?

Reply:
OK, changed now